docs/components/store.md

```diff
@@ -319,61 +319,3 @@ Following options are used for metadata caching (meta.json files, deletion mark
 - `metafile_max_size`: maximum size of cached meta.json and deletion mark file. Larger files are not cached.
 
 Note that chunks and metadata cache is an experimental feature, and these fields may be renamed or removed completely in the future.
-
```

Comment:
Can we link those two pages at least? :hugs:

Reply:
Done. What do you think?